api/v1beta1/verticarestorepointsquery_types.go

```diff
@@ -47,8 +48,54 @@ const (
 // VerticaRestorePointsQueryStatus defines the observed state of VerticaRestorePointsQuery
 type VerticaRestorePointsQueryStatus struct {
 	// +operator-sdk:csv:customresourcedefinitions:type=status
-	// RestorePoints used to list out the available restore points.
-	RestorePoints string `json:"restorePoints"`
+	// Conditions for VerticaRestorePointsQuery
+	Conditions []VerticaRestorePointsQueryCondition `json:"conditions,omitempty"`
```

Comment:
We should use the `metav1.Condition` type rather than creating our own. This allows us to use the condition functions in the operator-sdk package. We use this in the v1 VerticaDB API (search for metav1.Condition).